iidm/iidm-converter-api/src/test/java/com/powsybl/iidm/import_/ImportersTest.java

```diff
@@ -209,4 +215,27 @@ public void loadNetworks() throws InterruptedException, ExecutionException, IOEx
         assertEquals(2, isLoadPresent.size());
         isLoadPresent.forEach(Assert::assertTrue);
     }
+
+    @Test
+    public void postProcessorWithReporter() throws IOException {
+        ImportPostProcessorMock importPostProcessorMock = new ImportPostProcessorMock();
+        ImportersLoader loader = new ImportersLoaderList(Collections.singletonList(testImporter), Collections.singletonList(importPostProcessorMock));
+        Importer importer1 = Importers.addPostProcessors(loader, testImporter, computationManager, "testReporter");
+
+        ReporterModel reporter = new ReporterModel("testPostProcessor", "Test importer post processor");
+        Network network1 = importer1.importData(null, NetworkFactory.findDefault(), null, reporter);
+        assertNotNull(network1);
+
+        Optional<Report> report = reporter.getReports().stream().findFirst();
+        assertTrue(report.isPresent());
+
+        StringWriter sw = new StringWriter();
+        reporter.export(sw);
```

Comment:
It's more consistent to check the serialization, you can use `ReporterModelSerializer::write` for that.